Sources/SPMTestSupport/Concurrency.swift

```diff
@@ -0,0 +1,31 @@
+//
```

Comment:
This file needs to use a swift.org license header instead, as do other files in the package.

Reply:
You also need to add it to `Sources/SPMTestSupport/CMakeLists.txt` for bootstrapped builds to pass on CI.

Reply:
Also, if you address the rest of the feedback on this PR, you probably won't need to define `safe_async` and this file won't be needed either.

Reply:
The CMakeLists.txt file does not exist anymore?

Reply:
You're right, turns out the reason for bootstrapped builds failure is the lack of `import _Concurrency` in this new file.

Reply:
the file is removed show it should be good to go